omnibus/config/projects/agent.rb

```diff
@@ -181,3 +181,9 @@
 
 exclude '\.git*'
 exclude 'bundler\/git'
+
+# removing the man pages from the embedded folder to reduce package size by ~4MB
+if not windows? # linux + osx
+  FileUtils.rm_f("#{install_dir}/embedded/man")
+  FileUtils.rm_f("#{install_dir}/embedded/share/man")
```

Comment:
I don't think this'll actually work since this will run during the omnibus compilation step, instead of at the omnibus build step.

You could either use the `exclude` DSL method, or put this in the `datadog-agent-finalize` step to keep logic out of the project file as much as possible.

Long-term, what do you think about putting this "cleanup" logic directly in the software definitions that actually create these files? That approach would be more work initially, but to me it fits better with the "omnibus way" of having self-contained software definition (see my comment about that on the initial trello card as well).

Reply:
We could definitely do this, that's a bit of work on `omnibus-ruby`'s side, but for example we could have a `cleanup` DSL method that behaves the same as the `build` DSL method. All `cleanup` blocks would then be executed after the `build` blocks have all been executed.